PUBLISH.md

```diff
@@ -0,0 +1,42 @@
+# Publish guide
+
+The `plaid-node` library is published via npm at [npmjs.com/plaid][1].
+
+1. **Install dev-dependencies**
+
+Make sure you have all of your dev dependencies installed before releasing a
+new version of `plaid-node`:
+
+```bash
+make setup
+```
+
+2. **Publish and release to NPM**
+
+Creating a new release and publishing to npm is simple and bundled into a
+single make command.
+
+Use semantic versioning to determine whether a release should be one of the
+follow version bumps:
+- `patch`
+- `minor`
+- `major`
+
+```bash
+make release-(patch|minor|major)
+```
+
+Run the following git command to ensure the new version tag is pushed to the
+`plaid-node` GitHub repository:
+
+```bash
+git push --follow-tags
```

Comment:
IIRC this was only required for plaid-go and not plaid-node. xyz takes care of pushing git tags and branches directly to master.

Reply:
⚡️